test/library/helpers.go

```
@@ -0,0 +1,320 @@
+		out, err := oc.AsAdmin().Run("process").Args(parameters...).Output()
+		if err != nil {
+			return false, nil
+		}
+		tmp, err := os.CreateTemp("", "kcm-workload-*.json")
+		if err != nil {
+			return false, err
+		}
+		if _, err := tmp.WriteString(out); err != nil {
+			return false, err
+		}
+		tmp.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `tmp.Close` is not checked

(errcheck)